### PR TITLE
Replace request with axios 

### PR DESCRIPTION
**Pull request by Dexus**
based on #166

### CODE REVIEW

**Review by abebeos — APPROVED** [review]
approving without real review/app-testing

**Comment by abebeos** [comment]
@Dexus , again, you can merge this one yourself after #166

File: main/index.html [hunk 1]
```diff
@@ -627,43 +627,36 @@
 						url = defaultConversionServer;
 					}
 					
-					var req = request.post(url, function (err, resp, body) {
-						if (err) {
-							message('could not contact file conversion server', true);
+					axios.post(url, { format: 'svg', fileUpload: fs.createReadStream(file) }).then(body => {
```
Comment (ShaMan123) [inline main/index.html:630]: The format seems off... image/svg+xml?

**Comment by ShaMan123** [comment]
I wanted to understand this part... Why is the file converting service not part of the app? It can be exposed by the main module... From a server perspective it is not desirable. I eould expect to be able to deploy Deepnest to a server with this ability and without networking (important for priducts that need strict data compliance)

File: main/index.html (hunk 1 above)
Comment (abebeos) [reply main/index.html:630]: @Dexus , take a look. I'll publish a electron 31 version so users ca do the testing. most possibly tomorrow.

**Comment by abebeos** [comment]
> Why is the file converting service not part of the app?

the original author keeps non-public code on a server.

**Comment by ShaMan123** [comment]
> > Why is the file converting service not part of the app?
> 
> the original author keeps non-public code on a server.

So eventhough it is open for public use it is not open source? Is it open for public use?

**Comment by abebeos** [comment]
> So eventhough it is open for public use it is not open source? Is it open for public use?

Something like this. Original author is unreachable since years, but server seems operating stable (paywall was introduced, but seemingly removed again).

For me (and others) to find peace, deepnest needs to get decoupled from this server.

* see additionally https://github.com/deepnest-io/Deepnest/issues/165

File: main/index.html (hunk 1 above)
Comment (Dexus) [reply main/index.html:630]: Format was for the online converter to know in which format it should convert. So its a form field here like the origin code.

Comment (ShaMan123) [reply main/index.html:630]: Got it!

**Comment by Dexus** [comment]
I think it is. Every file that was dxf was send to the server. And every SVG result to dxf also on export.

**Comment by abebeos** [comment]
@Dexus , I removed this PR by force-pushing it, see additionally:

* #176

**Pull request merged**